proto/frequenz/api/common/v1/metrics/metrics.proto

```diff
@@ -172,6 +172,49 @@ enum Metric {
   METRIC_SENSOR_IRRADIANCE = 166;
 }
 
+// Enumerated categories of connections from which metrics can be obtained for a
+// given electrical component or sensor.
+enum MetricConnectionCategory {
+  // Default value.
+  METRIC_CONNECTION_CATEGORY_UNSPECIFIED = 0;
+
+  // Represents a connection to a battery.
+  METRIC_CONNECTION_CATEGORY_BATTERY = 1;
+
+  // Represents a connection to a PV array.
+  METRIC_CONNECTION_CATEGORY_PV_ARRAY = 2;
+
+  // Represents a connection to a temperature sensor measuring the ambient
+  // temperature.
+  METRIC_CONNECTION_CATEGORY_AIR = 10;
```

Comment:
Also if this means it comes from the "ambient", "AIR" sounds a bit too vague, wouldn't it make more sense to have something maybe like `AMBIENT`? `EXTERNAL_ENVIRONMENT`?

Reply:
I changed it to `AMBIENT`.